dotlottie/src/main/java/com/lottiefiles/dotlottie/core/compose/ui/DotLottieAnimation.kt

```diff
@@ -0,0 +1,177 @@
+package com.lottiefiles.dotlottie.core.compose.ui
+
+import android.graphics.Bitmap
+import android.view.Choreographer
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import com.dotlottie.dlplayer.Mode
+import com.lottiefiles.dotlottie.core.model.Config
+import com.lottiefiles.dotlottie.core.util.DotLottieUtils
+import androidx.compose.runtime.*
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asImageBitmap
+import com.dotlottie.dlplayer.DotLottiePlayer
+import com.lottiefiles.dotlottie.core.compose.runtime.DotLottieController
+import com.dotlottie.dlplayer.Config as DLConfig
+import com.lottiefiles.dotlottie.core.util.DotLottieEventListener
+import com.sun.jna.Pointer
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.isActive
+import java.nio.ByteBuffer
+
+@Composable
+fun DotLottieAnimation(
+    modifier: Modifier = Modifier,
+    width: UInt,
+    height: UInt,
+    src: String = "",
+    asset: String = "",
+    data: Any? = null,
+    autoplay: Boolean = false,
+    loop: Boolean = false,
+    useFrameInterpolation: Boolean = true,
+    speed: Float = 1f,
+    playMode: Mode = Mode.FORWARD,
+    controller: DotLottieController = DotLottieController(),
+    eventListeners: List<DotLottieEventListener> = listOf(),
```

Comment:
I'm unaware of performance differences but, using emptyList() might make more semantic sense for default parameter init

Reply:
Thanks. I will update this.